src/index.html

```diff
@@ -7,8 +7,82 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html" class="logo">
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="logo__img"
+          >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#apple"
+              >Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#samsung"
+              >Samsung
+            </a>
+          </li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a
              class="nav__link is-active"
              href="#apple"
            >
              Apple
            </a>
          </li>
          
          <li class="nav__item">
            <a
              class="nav__link"
              href="#samsung"
            >
              Samsung
            </a>
          </li>
```

Add blank line betwen multiple sibling blocks of html and fix formatting

Reply:
done